plugins/ci/skills/list-symptoms/list_symptoms.py

```
@@ -0,0 +1,118 @@
+import urllib.request
+
+BASE_URL = "https://sippy.dptools.openshift.org/api/jobs"
+
+
+def item_url(resource, item_id):
+    """Build the URL for a single symptom/label, URL-encoding the ID."""
+    return "%s/%s/%s" % (BASE_URL, resource, urllib.parse.quote(item_id, safe=""))
+
+
+def fetch(url):
+    req = urllib.request.Request(url, headers={"Accept": "application/json"})
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)